cpp/include/cudf/io/parquet.hpp

```diff
@@ -99,6 +98,8 @@ class parquet_reader_options {
   bool _use_arrow_schema = true;
   // Whether to allow reading matching select columns from mismatched Parquet files.
   bool _allow_mismatched_pq_schemas = false;
+  // Whether to allow reading non-existent projected columns
+  bool _allow_missing_columns = false;
```

Comment:
with false as the default, this is still a breaking change. I think we should either default to true or move this PR to 25.12 and aim to merge early to allow users to adapt. Thoughts?